transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -275,6 +328,8 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
     val metadataFiles = getReadableFiles(s"$inputPrefix/metadata/${entityType}/**.json", context)
     val metadataFilenameAndMsg = jsonToFilenameAndMsg(entityType)(metadataFiles)
 
+    validateJson(metadataFilenameAndMsg)
```

Comment:
This will probably work in practice, but it isn't _technically_ the right control-flow. Calling this as a `Unit` method here registers a side-branch of processing in the graph. It's completely disconnected from the steps below, so when the pipeline is sent to Dataflow it'll be run in parallel with processing, instead of in sequence.

The fix is to have `validateJson` return an `SCollection`, and use that output as inputs downstream.